internal/tools/semconv-gen/generator.go

```diff
@@ -0,0 +1,300 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	flag "github.com/spf13/pflag"
```

Comment:
Do we really need to use https://github.com/spf13/pflag ? 
I feel that it is feature creeping and we would need to update this dependency. 
I would prefer to use just `flag`. 
Also, it is safer to just use the standard library from a security standpoint.

Reply:
It's an internal tool that doesn't ship in any end-user code.  I prefer the POSIX-style flags.  The pflag package is stable and widely used, I see no issue with using it here.